examples/versions.tf

```diff
@@ -11,7 +11,7 @@ terraform {
   required_providers {
     gitlab = {
       source  = "gitlabhq/gitlab"
-      version = "3.11.1"
+      version = "~> 15.0"
```

Comment:
This PR still updates gitlab provider version. Could you please ensure it updates terraform version only?

Reply:
Sorry I missed those, branch confusion :)
Should be good now.